src/commands/codepush/deployment/history.ts

```diff
@@ -0,0 +1,48 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../../util/commandline";
+import { out } from "../../../util/interaction";
+import { DefaultApp } from "../../../util/profile";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../../util/apis";
+import { formatDate } from "./lib/date-helper";
+const _ = require("lodash");
+const chalk = require("chalk");
+const format = require('date-fns/format');
```

Comment:
No mention of `date-fns` in the package.json file. Did you forget to add it? Also, don't use both moment and date-fns, please pick one.